ydb/library/yql/sql/v1/format/sql_format.cpp

```diff
@@ -55,7 +55,7 @@ bool Validate(const TParsedTokenList& query, const TParsedTokenList& formattedQu
             if (in->Name != out->Name) {
                 return false;
             }
-            if (AsciiEqualsIgnoreCase(in->Name, in->Content)) {
+            if (const bool isKeyword = AsciiEqualsIgnoreCase(in->Name, in->Content)) {
```

Comment:
After I set `NULL` suffix in any way, tests failed. I spent some time figuring out a cause of it. Added this variable to make it more clear for reader to understand what does this check.